everrest-core/src/main/java/org/everrest/core/servlet/ServletContainerRequest.java

```diff
@@ -44,15 +47,31 @@ public static ServletContainerRequest create(final HttpServletRequest req) {
             if (port < 0) {
                 port = forwardedUrl.getDefaultPort();
             }
+            if (LOG.isInfoEnabled()) {
+                LOG.info("Assuming forwarded URL: " + forwardedUrl);
```

Comment:
you can replace this block with

 LOG.debug("Assuming forwarded URL: {}" , forwardedUrl);